WordPress/Classes/Models/Post.swift

```diff
@@ -195,27 +195,6 @@ class Post: AbstractPost {
 
     // MARK: - AbstractPost
 
-    override func hasSiteSpecificChanges() -> Bool {
```

Comment:
Do you think removing this might change some behaviour of other superclass methods in this class?

Reply:
It's a big one, thanks for the catch 💯 👏 
Actually `hasLocalChanges` of **super** uses this method

Reverted at 43d4abeca8e5f62b4838283b890a867f66b06ad3